.gitignore

```diff
@@ -18,7 +18,6 @@ lib
 lib_managed
 log
 logs
-modules
```

Comment:
leave these in for now, it does no harm and will prevent anyone with an existing repo from pushing these up accidentally